pkg/apicast/apicast.go

```diff
@@ -67,9 +71,21 @@ func (a *APIcast) deploymentVolumeMounts() []v1.VolumeMount {
 		})
 	}
 
+	for _, customPolicySpec := range a.options.CustomPolicies {
+		volumeMounts = append(volumeMounts, v1.VolumeMount{
+			Name:      policyVolumeName(customPolicySpec),
+			MountPath: path.Join(CustomPoliciesMountBasePath, customPolicySpec.Name, customPolicySpec.Version),
+			ReadOnly:  true,
+		})
+	}
+
 	return volumeMounts
 }
 
+func policyVolumeName(spec appsv1alpha1.CustomPolicySpec) string {
+	return fmt.Sprintf("%s-%s", "policy", helper.DNS1123Name(spec.VersionName()))
```

Comment:
I am curious about this. What's the limitation here? do you have some link to the doc for the limitation?

Reply:
The volume name does not accept dots, `.`. Only DNS1123 compliant names. When not sanitized, I had an error.

```
// Volume represents a named volume in a pod that may be accessed by any container in the pod.   
type Volume struct {                                                                             
    // Volume's name.                                                                            
    // Must be a DNS_LABEL and unique within the pod.                                            
    // More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names 
    Name string `json:"name" protobuf:"bytes,1,opt,name=name"`                                   
```

Adding `policy` prefix to know more context about the volume when you list volumes of the Deployment resource.

Reply:
:+1: